features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2010-2018 by the respective copyright holders.
```

Comment:
Can you leave this alone and add your binding in this file

Reply:
I think `mvn license:format` got this file 🤦‍♂️